scripts/build-deb.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eux -o pipefail
+
+project_dir="$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)"
+version="$(git describe --tags --always --dirty)"
+
+echo "project root: $project_dir"
+echo "project version: $version"
+
+cd "$project_dir"
+ln -sf packaging/debian debian
+debuild
+
+mkdir -p out
+mv ../azure-nvme-utils_99.99.99* out/
```

Comment:
This glob misses `azure-nvme-utils-dbgsym_99.99.99*` packages.

Reply:
fixed, thanks!